rpc/verif_proxy_test.go

```diff
@@ -14,16 +14,15 @@ import (
 
 	"net/http"
 	"net/http/httptest"
-)
 
-import (
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/ethereum/go-ethereum/common/hexutil"
-	gethrpc "github.com/ethereum/go-ethereum/rpc"
-	"github.com/status-im/status-go/params"
 
 	"github.com/stretchr/testify/require"
 	"github.com/stretchr/testify/suite"
+
+	gethrpc "github.com/ethereum/go-ethereum/rpc"
+	"github.com/status-im/status-go/params"
```

Comment:
Looks like your editor has jumbled up the imports for some reason.

Reply:
interesting, that's the output of `make lint-fix`